dist/angular-semantic-ui.js

```diff
@@ -355,6 +359,12 @@ angular.module('angularify.semantic.dropdown', [])
         scope.item_value = scope.value;
       }
 
+      scope.active_class = function(value) {
```

Comment:
Can anyone please review this? Is creating a scope function in the linking process ok since this doesn't have a controller of its own?